src/tests/reflection/GenericAttribute/GenericAttributeTests.cs

```diff
@@ -22,6 +22,13 @@ static int Main(string[] args)
         Assert(!CustomAttributeExtensions.GetCustomAttributes(assembly, typeof(SingleAttribute<>)).GetEnumerator().MoveNext());
 */
 
+        // Module module = programTypeInfo.Module;
+        // AssertAny(CustomAttributeExtensions.GetCustomAttributes(module), a => a is SingleAttribute<long>);
+        // Assert(CustomAttributeExtensions.GetCustomAttributes(module, typeof(SingleAttribute<long>)).GetEnumerator().MoveNext());
+        // Assert(CustomAttributeExtensions.GetCustomAttributes(module, typeof(SingleAttribute<long>)).GetEnumerator().MoveNext());
+        // Assert(!CustomAttributeExtensions.GetCustomAttributes(module, typeof(SingleAttribute<>)).GetEnumerator().MoveNext());
+        // Assert(!CustomAttributeExtensions.GetCustomAttributes(module, typeof(SingleAttribute<>)).GetEnumerator().MoveNext());	
```

Comment:
Should we include these tests within the above mulitilne comment? Seems related to me

Reply:
I wasn't sure if it was related and wanted to ask about that. It's possible that these tests could be brought back.

Do you know what the preferred approach is for iterating on this particular test locally? [https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/coreclr/testing.md#build-individual-test](this page) suggests that I should run 

> dotnet.cmd build c:\dev\runtime\src\tests\reflection\GenericAttribute\GenericAttributeMetadata.ilproj -c Release

but that just builds a dll and doesn't seem to actually invoke this main method.

Reply:
Sorry idk how to run them locally, @davidwrighton @jkotas might help with that

Reply:
The instructions for running these tests are at https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/coreclr/windows-test-instructions.md